src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -84,8 +84,9 @@ public CoreNode(NodeRunner runner, NodeConfigParameters configParameters, string
             this.runner = runner;
 
             this.State = CoreNodeState.Stopped;
+            string user = Encoders.Hex.EncodeData(RandomUtils.GetBytes(20));
             string pass = Encoders.Hex.EncodeData(RandomUtils.GetBytes(20));
-            this.creds = new NetworkCredential(pass, pass);
+            this.creds = new NetworkCredential(user, pass);
```

Comment:
This is incorrect for StratisX nodes - it throws an error about how user and pass can't be the same.